packages/core/package.json

```diff
@@ -32,6 +32,7 @@
     "dev": "rollup -cw",
     "build": "rollup -c",
     "test": "vitest run",
+    "test:watch": "vitest watch",
```

Comment:
I couldn't get into `watch` mode without this, but I didn't mean to add it to the commit. Will drop.